spot-contracts/contracts/FeePolicy.sol

```diff
@@ -74,6 +69,9 @@ contract FeePolicy is IFeePolicy, OwnableUpgradeable {
     /// @notice Target subscription ratio higher bound, 2.0 or 200%.
     uint256 public constant TARGET_SR_UPPER_BOUND = 2 * ONE;
 
+    /// @notice The enforced maximum change in perp's value per rebalance operation.
+    uint256 public constant MAX_PERP_VALUE_CHANGE_PERC = ONE / 100; // 0.01 or 1%
```

Comment:
Do we need this? Could it get in the way in the future?

Reply:
Yea lets get rid of this